airline-web/public/css-templates/treviso/css/style.css

```diff
@@ -16,7 +16,7 @@ Designed & Handcrafted by Zan from ninetofive.me
 /******************************************************************
 	Fonts
 ******************************************************************/
-@import url(https://fonts.googleapis.com/css?family=Open+Sans:300,400,700);								
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@100..900&display=swap');								

```

Comment:
@cazh0 maybe just import like 300,400,700? I really doubt we're using any weights except normal & bold ha